client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -471,6 +485,15 @@ export default {
 
     ...mapMutations('SegmentFilter', ['updateFilterQuery']),
 
+    addClickListenerToImages () {
```

Comment:
since this get triggerted once in a while, we should remove the eventlisteners too. Otherwise we may got several listeners on the same element

Reply:
:bulb: